frameworks/kafka/tests/test_jmx.py

```diff
@@ -39,11 +39,11 @@ def install_jmx_configured_kafka(self_signed_trust_store: bool = True):
                                  "trust_store_password_file": TRUST_STORE_PASS}}},
             service_options)
 
-    sdk_install.install(
-        package_name=config.PACKAGE_NAME,
-        service_name=foldered_name,
+    config.install(
+        config.PACKAGE_NAME,
+        foldered_name,
+        1,
         additional_options=service_options,
-        expected_running_tasks=1
```

Comment:
Why this change? You removed named args for 3 out of 4. Why leave `additional_options`? But I would have argued to keep all the named arguments otherwise how would anyone know (without another hop) that `1` is for `expected_running_tasks`?